domains/certificates/Query.API/API/MeasurementsSyncer/SlidingWindowService.cs

```diff
@@ -27,19 +21,29 @@ public MeteringPointTimeSeriesSlidingWindow CreateSlidingWindow(Gsrn gsrn, UnixT
         return MeteringPointTimeSeriesSlidingWindow.Create(gsrn, synchronizationPoint, missingMeasurements);
     }
 
-    public List<Measurement> FilterMeasurements(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements)
+    public List<Measurement> FilterMeasurements(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements, int minimumAgeBeforeIssuingInHours)
     {
+        var threshold = UnixTimestamp.Now().Add(-TimeSpan.FromHours(minimumAgeBeforeIssuingInHours));
+
         return measurements
             .Where(m => m.Gsrn == window.GSRN)
             .Where(m =>
             {
                 if (m.QuantityMissing)
                 {
-                    _measurementSyncMetrics.AddFilterDueQuantityMissingFlag(1);
+                    measurementSyncMetrics.AddFilterDueQuantityMissingFlag(1);
                 }
                 return !m.QuantityMissing;
             })
             .Where(m =>
+            {
+                var dateTo = UnixTimestamp.Create(m.DateTo);
```

Comment:
So measurements are filtered here, how does this impact the sliding window and the synchronization point?